recipes/tokei/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "7.0.3" %}
+
+package:
+  name: tokei
+  version: {{ version }}
+
+source:
+  url: https://github.com/Aaronepower/tokei/releases/download/v{{ version }}/tokei-v{{ version }}-x86_64-unknown-linux-gnu.tar.gz  # [linux]
```

Comment:
Thanks for the submission.  
In general conda-forge prefers not to repackage binary artifacts unless there is no other way of building the package.  Could we try at least to build for osx and windows?

Reply:
Ok. will try that.